test/test-parse.cpp

```diff
@@ -1833,5 +1833,12 @@ TEST(test_parse, continue_statement) {
     EXPECT_THAT(v.visits, IsEmpty());
   }
 }
+
+TEST(test_parse, debugger_statement) {
+  {
+    spy_visitor v = parse_and_visit_statement(u8"debugger;");
+    EXPECT_THAT(v.visits, IsEmpty());
```

Comment:
Minor: Checking that `v.visits` is empty is a bit risky. What if the parser consumed no tokens at all? What if the parser didn't consume the semicolon?

One way we've handled this case before: add a canary statement at the end of the JavaScript snippet, parse two statements (instead of one), then assert that the canary statement was seen by the parser. See the `if (true) return; x;` test `test_parse.return_statement` for an example.

Testing this much might be more trouble than necessary. It's easy to go overboard with testing. =X